chart/templates/pgbouncer/pgbouncer-service.yaml

```diff
@@ -53,7 +53,7 @@ spec:
     - name: pgbouncer
       protocol: TCP
       port: {{ .Values.ports.pgbouncer }}
-    - name: pgbouncer-metrics
+    - name: pgb-metrics
```

Comment:
change here and also in the service because when install it I get and error that port name cannot be longer then 15 chars.

Reply:
Good observation